gatsby/src/components/Sidebar/index.js

```diff
@@ -1,7 +1,15 @@
-import React from "react";
+import React, { useState } from "react";
 import { graphql, StaticQuery, Link as GatsbyLink } from "gatsby";
 import styled from "styled-components";
-import { Text } from "grommet";
+import {
+  Text,
+  Box,
+  Accordion,
+  AccordionPanel,
+  ResponsiveContext,
+  Heading
+} from "grommet";
+import { FormDown, FormUp } from "grommet-icons";
 import { axisThemeConfig } from "@centrifuge/axis-theme";
```

Comment:
I would avoid using import { axisThemeConfig } from "@centrifuge/axis-theme";  in components. It creates a direct dependency and you can not override the styles if you wanna do that. props.theme should be axisThemeConfig here from the ThemeProvider